docs/operator-guides/v2-migration.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Blazar (v2) Migration
+sidebar_position: 6
+---
+
+# EigenDA Blazar (v2) Migration
+
```

Comment:
can we have a subsection to talk about migration timeline, node spec (same as v1 for now), and ejection policy?